src/Tgstation.Server.Host.Service/Program.cs

```diff
@@ -118,6 +118,33 @@ public async Task OnExecuteAsync()
 				{
 					if (Uninstall)
 						return; // oh no, it's retarded...
+
+					// First check if the service already exists
+					foreach (ServiceController sc in ServiceController.GetServices())
```

Comment:
You should do this in the `Environment.IsUserInteractive` block above., probably right after the prompt for if you wan to install it or not